styles.css

```diff
@@ -0,0 +1,89 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
+    align-items: center;
+    height: 40px;
+  }
+
+  section {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 10px;
+    padding: 20px;
+    background-image: url("images/background.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
```

Comment:
In order to have the background image responsive for different mobile versions and in order for your UI to look the same regardless of the mobile screen size, we need another setting for the background size.

- [ ] Please consider changing the background-size from `cover` to `100vw`.